demo_serial/src/main/java/com/novoda/downloadmanager/demo/QueryForDownloadsAsyncTask.java

```diff
@@ -0,0 +1,57 @@
+package com.novoda.downloadmanager.demo;
```

Comment:
or do both demos have the same java package? (and it's onlt the application ID you change?) because that is fine as well